docker-for-mac/faqs.md

```diff
@@ -184,7 +184,43 @@ For full information on adding server and client side certs, see [Adding
 TLS certificates](/docker-for-mac/index.md#adding-tls-certificates) in
 the Getting Started topic.
 
-### How do I reduce the size of Docker.qcow2?
+### Disk Usage
+
+#### Qcow2 or Raw?
+
+By default Docker for Mac stores containers and images in a file
+`Docker.raw` or `Docker.qcow2` in the directory
+`~/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux`.
+Starting with High Sierra with APFS (Apple Filesystem) enabled, Docker
+uses the "raw" format (`Docker.raw`), otherwise it uses the Qcow2
+format (`Docker.qcow2`).
+
+#### `Docker.raw` consumes an insane amount of disk space!
+
+This is an illusion.  Docker uses the raw format on Macs running the
+Apple Filesystem (APFS).  APFS supports sparse files, i.e., files
+where long series of "zeros" are compressed.  Calling `ls` is
+misleading, because it reveals the logical size of the file, not its
+physical size; be sure to use `-s` to display the physical size:
+
+```bash
+..ker/Data/com.docker.driver.amd64-linux $ ls -klsh Docker.raw
```

Comment:
Simplify the prompt to just `$`

Also you don't need all these flags, just use `-k` or `-h`. Maybe explain that `-l` allows the long listing and `-h` uses a human-readable size.

Reply:
Hi.

My point was to show the directory we're in.  So I guess I'll add a `cd` before.

I can sure teach the meaning of the flags, but I believed that what matters is how it's displayed, and this is not a documentation for `ls`.